server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/MnemonicName.java

```diff
@@ -32,9 +32,9 @@
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 public class MnemonicName extends AbstractVariable {
-  private String systemName;
-  public MnemonicName(String systemName, String name, Locality definition) {
-    super(-1, name, definition, null);
+  String systemName;
+  public MnemonicName(String systemName, String name, Boolean global, Locality definition) {
```

Comment:
I think this is always false by defintion, so no need to accept it as a constructor parameter

Reply:
Agree.